server/src/main/java/io/crate/fdw/ServersMetadata.java

```diff
@@ -344,4 +346,11 @@ public Iterable<Option> getOptions() {
             .flatMap(server -> server.getOptions())
             .iterator();
     }
+
+    public List<String> getServerNamesOwnedBy(String owner) {
```

Comment:
Could instead of exposing a extra property iterate over the servers metadata directly. It's an Iterable of Server. The one consumer is only interested if at least one server owned by the user exists and doesn't need a list

Reply:
I was trying to return the list of servers under the user in the exception message, but I still made the change, the method seemed too specific.